e2e/support/config.js

```diff
@@ -22,6 +22,7 @@ const sourceVersion = process.env["CROSS_VERSION_SOURCE"];
 const targetVersion = process.env["CROSS_VERSION_TARGET"];
 
 const runWithReplay = process.env["CYPRESS_REPLAYIO_ENABLED"];
+const uploadReplayRecordings = process.env["CYPRESS_REPLAYIO_ENABLE_UPLOAD"];
```

Comment:
I would suggest removing `CYPRESS_REPLAYIO_ENABLE_UPLOAD: 0` from the other file and combining it with this change:
```suggestion
const uploadReplayRecordings = !!process.env["CYPRESS_REPLAYIO_ENABLE_UPLOAD"];
```